app/pods/components/pagination-controls/template.hbs

```diff
@@ -1,2 +1,4 @@
-<button {{action 'prev'}}>Previous 20</button>
-<button {{action 'next'}}>Next 20</button>
+<div class="pagination">
```

Comment:
Ember will already wrap this partial in a `div`. You can specify classes to add to the div via the `classNames` property on the component itself.

https://guides.emberjs.com/v2.4.0/components/customizing-a-components-element/#toc_customizing-class-names